docs/pages/attributes.rst

```diff
@@ -0,0 +1,215 @@
+  
+"up"
+	- 3-vector
+	- Describes the **up** direction for the scene.
+"front"
+	- 3-vector
+	- Describes the **forward** direction for the scene.
+"origin"
+	- 3-vector
+	- Describes the **origin** of the scene, for alignment purposes.
+	 
+Scene Physics and Object-related Parameters
+-------------------------------------------
+
+Below are scene-specific physical and object-related quantities.  These values will override similarly-named values specified in the Physics Manager Attributes.
+
+"scale"
+	- 3-vector
+	- The default scale to be used for the scene.
+"gravity"
+	- 3-vector
+	- Gravity to use in scene for physical modeling.
+"margin"
+	- double
+	- Distance margin for collision calculations.
+"friction coefficient"
+	- double
+	- The coefficient of friction.
+"restitution coefficient"
+	- double
+	- The coefficient of restitution.
+"units to meters"
+	- double
+	- The conversion of given units to meters.
+"requires lighting"
+	- boolean
+	- Whether or not the scene should use lighting.
+"lighting setup"
+	- string
+	- The Habitat lighting setup to use if lighting is required.
+"rigid object paths"
+	- list of strings
+	- A list of locations to query for supported object files that should be available to be loaded in the scene.
+
+ 
+
+`Object Attributes`_
+====================
+Object Attributes templates hold descriptive information for type of object that can be loaded into Habitat.  These files should be named in the format : 
+
+ 	<objectname>.phys_properties.json
+
+Object Mesh Handles And Types
+-----------------------------	
+
+Below are the handles and descriptors for various mesh assets used by an object.
+	
+Mesh types are included to support future format flexibility. In the JSON they are limited to string keys used by :ref:`AbstractPhysicsAttributes::AssetTypeNamesMap`, which map to values in :ref:`AssetType`.  Any mesh type specification used in JSON but not found in :ref:`AbstractPhysicsAttributes::AssetTypeNamesMap` results in that mesh type being set to :ref:`AssetType::UNKNOWN`.  JSON-Specified mesh types will override any default settings, such as those that might be derived from file name extensions.
+
+
+"render mesh" 
+	- string
+	- The name of the file describing the render mesh to be used by the object.
+"render mesh type"
+	- string
+	- The type of the render mesh. If not specified, defaults to :ref:`AssetType::UNKNOWN`.
+"collision mesh"
+	- string
+	- The name of the file describing the collision mesh to be used by the object. 
+"collision mesh type"
+	- string
+	- The type of the collision mesh. If not specified, defaults to :ref:`AssetType::UNKNOWN`.
+	
+
+Object Frame and Origin
+-----------------------
+
+The tags below are used to build a coordinate frame for the object, and will override any default values set based on render mesh file name/extension.  If either **"up"** or **"front"** are specified, both must be provided and they must be orthogonal.  The object's COM is used as its origin.
```

Comment:
Beware, we're advising Appen to deliver objects with the origin placed at the COM projected down to ground height. In other words, we're going to have some objects soon where the origin is not the COM.

Reply:
> I left a bunch of minor comments just as feedback from a user perspective. Don't feel obligated to respond or make changes. Thanks for writing good documentation!

Thank you for being thorough Eric, I am glad you looked things over, and sorry I didn't include you as a reviewer. :)

Reply:
The COM is used as the origin to build the object's Coordinate frame (frame requires "up", "front" and "origin" for rotation calcs) - this origin field is specified above this section in the static scene discussion, so I mention this here to address the potential question "where is the origin?". 

The object COM can differ from the mesh COM, if the user wishes it - it is either specified by the user in json, or else computed when an object is built.